cmd/performance-profile-creator/cmd/root.go

```diff
@@ -83,14 +83,17 @@ func getDataFromFlags(cmd *cobra.Command) (profileCreatorArgs, error) {
 	creatorArgs := profileCreatorArgs{}
 	mustGatherDirPath := cmd.Flag("must-gather-dir-path").Value.String()
 	mcpName := cmd.Flag("mcp-name").Value.String()
+	log.Infof("mcp-name: %s ", mcpName)
```

Comment:
could you please add in the commit message why/how this change improve the log messages?
Also, should these logs be Info or should they be debug? Or do we want to emit them only if a certain verbosity level is given?

Reply:
We want to send to stderr as INFO the exact reason we created the profile so the created profile would not have "magic" results. We intend to justify each field in the profile we create. E.g. the node selector comes from the mcp node selector.

Reply:
Sure, and so verbose logging is always warranted? Shouldn't the user opt in?

Reply:
Additionally: I see some merit in (optionally) logging the compute field. But what do we gain in echoing back the value given as options?

Reply:
Echoing the input value does no help, indeed. Listing the nodes belonging to the mcp will help.
Until now it was to early to have a decision on "logging" levels, I thought "showing" the user how we compute the values is not "verbose mode", but a way to expose the user how we do things.
If you think this should not be the default option, we need a new command line arg.

Reply:
Will update to remove the input values and update add log for listing the nodes belonging to the mcp (missed that).

Reply:
OK, if we just log the computed field we can leave on as default and add a verbose option later.

Reply:
Ack.